vip-integrations.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * Plugin Name: VIP Integrations
+ * Description: Plugin loading for VIP plugin integrations.
+ * Author: Automattic
+ * License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
+ */
+
+namespace Automattic\VIP\Integrations;
+
+// New integrations names go here
+const BLOCK_DATA_API_SLUG = 'block-data-api';
```

Comment:
We can't use an enum here (beucase it's PHP 8+) but maybe a stub class would make sense?

```
class Integration {
    const BLOCK_DATA_API = 'block-data-api';
    // in the future, we can add wp-parsely and others here.
}
```

The usage becomes a little cleaner:

```
load_integration( Integration:BLOCK_DATA_API );
```

Reply:
Good point, I turned it into an abstract class and used that instead.